distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalogSearchUi.java

```diff
@@ -106,9 +109,7 @@ public void beforeExam() throws Exception {
 
     @After
     public void cleanUp() {
-        ids.stream()
-                .forEach(TestCatalogSearchUi::delete);
-        ids.clear();
+        console.runCommand(REMOVE_ALL);
```

Comment:
Also wouldn't be unreasonable to provide a specialized `clearCatalog` method on `AbstractIntegrationTest` that invokes its own `runCommand`.